src/App.tsx

```diff
@@ -8,7 +8,61 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterOption } from './enums/filter-options';
+import { getFilteredTodos } from './components/helpers/getFilteredTodos';
+import { getTodosByQuery } from './components/helpers/getTodosByQuery';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [searchQuery, setSearchQuery] = useState<string>('');
+  const [filterOption, setFilterOption] = useState<FilterOption>(
+    FilterOption.all,
+  );
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+
+  const handleFilterTodos = (event: ChangeEvent<HTMLSelectElement>) => {
+    setFilterOption(event.target.value as FilterOption);
+  };
```

Comment:
The method name 'handleFilterTodos' does not follow the naming conventions suggested in the checklist. It would be better to name it 'onFilterOptionChange' as it describes more accurately what the function does.